src/filters/common.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
```

Comment:
Missing `@module` definition. Also, I think we can mark it as `@protected`.

Reply:
Besides, we usually call such modules `utils.js`. Although, since those 3 functions here do the base preparations, I'd consider calling them `init.js`, `base.js`, `preprocessing.js`, etc.

Reply:
Another thing – what's the chance that these functions will be ever used (not counting tests) alone, not all at once? Isn't the only use case for them being called together so to create a `data[body/styles]` object? If so – a single `parse()` function would do better. It can do all those things at once. You can still export utils functions but then only for testing purposes. 

By focusing on one function you limit the API surface and make the code less prone to changes.

Reply:
The idea was that for content pasted from Word you need to extract and parse the body and do same for CSS. For content from google docs you just need to parse the input as a `body` because there is not `head` or `style` tag. For input from another text editor you may need other combination of functions to extract what is needed to properly transform the content. 

So it really depends on an input. Ofc `parse()` function may be created in such way that it checks what elements are in the provided input and parses them accordingly. Or for each type of input, different `parse()` function can be used.

> Besides, we usually call such modules utils.js. Although, since those 3 functions here do the base preparations, I'd consider calling them init.js, base.js, preprocessing.js, etc.

Do you mean to put each function in a separate file? And then create a `parse()` function which will import and use them?